jobs/resource_job.go

```diff
@@ -446,6 +446,45 @@ type JobsAPI struct {
 	context context.Context
 }
 
+// JobCreate + JobSettingResource related aliases.
+var jobsAliases = map[string]map[string]string{
+	"jobs.JobSettingsResource": {
+		"tasks":        "task",
+		"parameters":   "parameter",
+		"job_clusters": "job_cluster",
+		"environments": "environment",
+	},
+	"jobs.JobCreateStruct": {
+		"tasks":        "task",
+		"parameters":   "parameter",
+		"job_clusters": "job_cluster",
+		"environments": "environment",
```

Comment:
Since these are repeated, we could have them in a single map to reduce redundancy.